src/stravalib/tests/unit/test_protocol_auth.py

```diff
@@ -0,0 +1,70 @@
+from unittest.mock import patch
```

Comment:
Tests to ensure a return of either 1 or 2 items is provided. We could use this same structure to test refresh token IF it were useful.